OpenRA.Mods.Common/Traits/Player/PlaceBuilding.cs

```diff
@@ -16,10 +16,18 @@
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Allows to execute build orders.", " Attach this to the player actor.")]
-	class PlaceBuildingInfo : TraitInfo<PlaceBuilding> { }
+	class PlaceBuildingInfo : ITraitInfo
+	{
+		[Desc("Name of the palette to display the placement sprite in.")]
```

Comment:
The english here feels a bit off.  I suggest "Palette to use for rendering the placement sprite."